terraform/groups/document-signing-api/data.tf

```diff
@@ -9,7 +9,8 @@ data "aws_iam_policy_document" "image_bucket" {
     ]
 
     resources = [
-      "arn:aws:s3:::${var.image_bucket_name_prefix}-*"
+      "arn:aws:s3:::${var.image_bucket_name_prefix}-*",
+      "arn:aws:s3:::ch-${var.image_bucket_name_prefix}-*"
```

Comment:
The changes here will result in an unnecessary resource in the policy which could be cause for confusion—`document-api-images-*` in the `staging` account won't match any resources, and similarly `ch-document-api-images-*` in the `development` account won't match any resources[^1].

Revert these changes:

```suggestion
      "arn:aws:s3:::${var.image_bucket_name_prefix}-*"
```

Then, override the `image_bucket_name_prefix` variable default value for the `staging` environment in the `groups/document-signing-api/profiles/staging-eu-west-2/vars` file:

```terraform
image_bucket_name_prefix = "ch-document-api-images"
```

[^1]: A single bucket in the `development` account _does_ include the `ch-` prefix, however this should be ignored as it was created for testing iXBRL and is outdated.

Reply:
Thank you @marcransome for explaining this. I have made the changes.